### PR TITLE
Add NewRelic integration for bulk-index

### PR DESCRIPTION
**Pull request by nonword**
Adds basic NewRelic integration in bulk-index script, including encrypted NR license key.

Updated README to document bulk-index usage generally, and also how to enable NR reporting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,3 +30,20 @@ To run lambda locally, use this script:
...
 
+## Running
+
+### Bulk Indexing
+
+The bulk-indexer can be used to index lots of records at once by direct BibService SQL query.
+
+For example, to reindex the first 10K NYPL bibs with marc 001 in QA:
+```
+node scripts/bulk-index.js --envfile config/qa-bulk-index.env --type bib --hasMarc 001 --nyplSource sierra-nypl --limit 10000
```
Comment (charmingduchess) [inline README.md:41]: 👏🏻 incredible!! did we have this script anywhere before?

File: scripts/bulk-index.js [hunk 2]
```diff
@@ -252,62 +260,74 @@ const printProgress = (count, total, startTime) => {
...
+  })
 
   const total = argv.limit
   let count = 0
   const startTime = new Date()
   while (count < argv.limit || !argv.limit) {
-    // Log out progress os far:
-    printProgress(count, total, startTime)
+    await newrelic.startBackgroundTransaction('Bulk-index batch', async () => {
+      // Log out progress os far:
```
Comment (charmingduchess) [inline scripts/bulk-index.js:309]: typo!

File: scripts/bulk-index.js [hunk 3]
```diff
@@ -252,62 +260,74 @@ const printProgress = (count, total, startTime) => {
...
+      printProgress(count, total, startTime)
```
Comment (charmingduchess) [inline scripts/bulk-index.js:310]: So if there was 1000 bibs getting reindexed, would this print 1000 times? Maybe we want to have a max number of prints, like 100 or so, to keep logs legible. eg, for bib count of 100 , printProgress might happen for every 10 bibs, for 1000, every 100.

Comment (nonword) [reply scripts/bulk-index.js:310]: This call is made [on each batch](https://github.com/NYPL/research-catalog-indexer/blob/95f28a4f00cbc88a29e89a12758b641dafc88612/scripts/bulk-index.js#L313) so is fired every `argv.batchSize` records ([default 100](https://github.com/NYPL/research-catalog-indexer/blob/95f28a4f00cbc88a29e89a12758b641dafc88612/scripts/bulk-index.js#L46))

File: README.md (hunk 1 above)
Comment (nonword) [reply README.md:41]: This is added documentation, but the functionality has been hanging out in [this related PR](https://github.com/NYPL/research-catalog-indexer/pull/71) waiting to merge to main

**Pull request merged**